src/services/request-limit-record-service.js

```diff
@@ -0,0 +1,50 @@
+import Logger from '../common/logger.js';
+import { createShortHash } from '../utils/string-utils.js';
+import RequestLimitRecordStore from '../stores/request-limit-record-store.js';
+
+const logger = new Logger(import.meta.url);
+
+const FAILED_LOGIN_ATTEMPTS_THRESHOLD = 5;
+const FAILED_LOGIN_ATTEMPTS_KEY = 'failed-login-attempts';
+const FAILED_LOGIN_ATTEMPTS_TIME_WINDOW_IN_MS = 15 * 60 * 1000;
+
+export default class RequestLimitRecordService {
+  static get inject() {
+    return [RequestLimitRecordStore];
+  }
+
+  constructor(requestLimitRecordStore) {
+    this.requestLimitRecordStore = requestLimitRecordStore;
+  }
+
+  getRoomById(roomId) {
+    return this.roomStore.getRoomById(roomId);
+  }
```

Comment:
This is probably a leftover.

Reply:
:scream: You should have never seen that.

Reply:
😵 🙈